interface/portraits/portraits_highelf.gfx

```diff
@@ -214,21 +226,21 @@ spriteTypes = {
 		effectFile = "gfx/FX/portrait.lua"
 		layer = { # GFX_TYPE:[d|p]INDEX:COLOR_LINK:DONT_REFRESH_IF_VALID:CULTURE_INDEX
 			"GFX_character_background:p0"
-			"GFX_empty:p3"
-			"GFX_empty:p5"
+			"GFX_highelf_female_clothes_behind:p3"
+			"GFX_highelf_female_headgear_behind:p5"
 			"GFX_empty:p1:h:y"
 			":p5:c6"
 
 			"GFX_highelf_female_base:p2"
 			"GFX_empty:d0"
-			"GFX_empty:p3"
+			"GFX_highelf_female_clothes:p3"
 			"GFX_empty:d1"
-			"GFX_empty:d2"
+			"GFX_highelf_female_cheeks:d4:h"
```

Comment:
You should move the eyebrows above the hairs.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81193142-106e8400-8fcc-11ea-9b4d-ac8e7dd3df95.png)

</details>